### DISCUSSION TITLE
Suggestions for improving Troll warlord's extremely lackluster kit for a carry hero

### DISCUSSION THREAD

**Discussion by kyhmr** [ValveSoftware/Dota2-Gameplay#10967] [Ideas]
**Summary:**
Troll warlord is extremely hard-countered by disables while having no better carry performance than other carries as a trade-off. This makes him a very weak hero that is extremely frustrating to play.
Suggested changes:
Berzerker’s rage root: increase the root chance to 25/30/35/40%, while reducing the root duration to 0.4/0.6/0.8/1 seconds (Doubled trigger chance and halved duration).
Whirling Axes: please no more buffs to Whirling Axes as it could potentially turn him into a support instead of improving his carry performance.
Fervor: Fervor stacks are no longer removed by switching targets, instead if troll warlord hasn’t hit something for 3 seconds, all Fervor stacks are removed.
Battle Trance: Too easy to counter with disables or kiting abilities, extremely situational, duration needs to be increased to 10 seconds.

**Intro**
I’ve been trying to make troll warlord work in turbo and normal matches as a solo que for a while now, but it’s an incredibly frustrating experience. Considering I have 6k hours already in dota 2, you’d think I could make this hero work. 
I’m no stranger to tough challenges but troll warlord is a hero that has inferior carry potential compared to other carries and is unbelievably hard countered by disables. Yes, I tried the Aghanims build, but you can’t dispel silences, hexes or stuns, neither does that give him the DPS he desperately needs. 
And if you’d go BKB, every other melee carry has superior performance with a BKB, so that doesn’t really count.

**Great recent changes**
The new shard improved his farming performance, which is nice and solves one of his core issues, being unable to farm as fast as other carries. 

**Suggested changes**

**Berzerker’s rage**
His passive or toggle got a hefty nerf with the stun being removed, as that mainly carried his entire attack speed design. I can see why the developers decided to remove it, but he’s so incredibly bad now that I don’t think it is wrong to tweak this skill a bit. 
The root has a good duration, but it doesn’t really play into his attack speed design. Thus, I’d suggest increasing the root chance to 25/30/35/40%, while reducing the root duration to 0.4/0.6/0.8/1 seconds. 
This way the root is a lot more consistent to trigger, and if troll warlord has a high attack speed, it is a much bigger difference than if he attacks with a lower attack speed (shorter duration vs longer duration).

**Whirling Axes**
His throwing/spinning axes should not be buffed further, as that would turn him into a support. So, I will make not suggestions on making those any more useful.

**Fervor**
Fervor is an awful passive, and not even up to carry-standard. Fights are over too quickly for troll warlord to gain any use out of it. And enemies can use one kite or disable abilities to get out of dodge, meaning they prevent troll warlord from getting any use out of this ability, as it would reset his Fervor stacks.
To make this passive on par with the carry passives of other heroes I suggest removing the aspect that makes his stacks reset when he switches targets. 
Meaning Fervor only expires after not hitting something for 3 seconds. This change would also make troll warlord’s passive stand apart more from ursa’s passive, and would mean that as long as troll warlord is hitting something, he becomes more dangerous, which neatly synergises with his ranged/melee toggle to keep hitting things.

**Battle Trance**
Battle trance is a pretty terrible ultimate in my opinion. It is a very situational spell that is designed to be used when troll warlord has taken significant damage. However, it can be easily countered by a disable. Not just to prevent him from casting it on time, but also while he is using it, at a 6.5 second duration, enemies only need two disables to prevent troll warlord from getting any use out of this ability. Or just one and kill him before he can activate it. Not to mention simply kiting him makes this ability useless as well.
Adding a spell immunity effect or status resistance would make it look too similar to ursa’s ultimate and lifestealer’s rage. So, I’d avoid doing that. 
Another option is having it auto-cast on 1 hp. But that would make it too similar to abaddons ultimate.
Thus, increasing the duration of the ability to 10 seconds seems like the fairest solution.

**Final word**
To anyone who thinks my assessment is wrong, please try troll warlord yourself and go down the same road of suffering I have. It could be that I got unlucky matched, but a person can only see his entire hero be destroyed by a few disables so many times before losing his mind. And when you can finally hit things, it isn’t all that impressive compared to other carries. Or enemies play incredible kiting spells to prevent you from doing the only thing your entire hero revolves around, hitting things.

**Comment by timsar2** [comment]
WK with aghanim is the problem. If troll kill one with in wk aghanim, The target will not change on any alive hero during the battle trance time.
